lib/salestation/web/extractors.rb

```diff
@@ -197,12 +197,28 @@ def self.extract(filters, request_hash)
             when Symbol
               stringified_key = filter.to_s
               extracted_data[filter] = request_hash[stringified_key] if request_hash.key?(stringified_key)
+              if extracted_data[filter].is_a?(Array)
```

Comment:
would calling
```
input = extract(filters, request_hash).deep_symbolize_keys
```
on line 189 also work instead of manually symbolizing the keys here?

Reply:
`deep_symbolize_keys` is a rails function. This library doesn't have access to rails as I know.

Reply:
we get it from `activesupport` library usually, but it's indeed not included here. so nvm